src/store/SettingsDialogStore.js

```diff
@@ -21,13 +22,39 @@ export default class SettingsDialogStore {
         this.onDeleted = onDeleted;
         this.menu = new MenuStore(mainStore, { route:'indicator-setting' });
         this.SettingDialogMenu = this.menu.connect(Menu);
+        reaction(() => (this.scrollPanel && this.open), () => {
+            if (!this.scrollPanel || !this.open) { return; }
+
+            const rootEle = this.scrollPanel.container;
+            const dropdowns = rootEle.querySelectorAll('.sc-color-picker, .sc-dropdown');
+            rootEle.addEventListener('click', () => {
+                this.setFreezeScroll(false);
+            });
+            dropdowns.forEach((dropdown) => {
+                dropdown.addEventListener('click', () => setTimeout(() => this.checkDropdownOpen(), 50));
+            });
+            this.setFreezeScroll(false);
+        });
     }
 
     get context() { return this.mainStore.chart.context; }
     get stx() { return this.context.stx; }
     get theme() { return this.mainStore.chartSetting.theme; }
 
     @computed get open() { return this.menu.open; }
+    @action.bound checkDropdownOpen() {
```

Comment:
I don't think we need `@action.bound` decorator here as we don't modify any observables

Reply:
@balakrishna-binary yes you are right. just modified and updated the test link